dbt/scripts/export_models.py

```diff
@@ -0,0 +1,212 @@
+# Export dbt models to Excel files.
```

Comment:
Note that this script is defined in a new `dbt/scripts/` subdirectory, rather than in `.github/scripts/` where `transform_dbt_test_results.py` is defined. I'm going to put up a PR soon to move `transform_dbt_test_results.py` into this directory since I've found that the location has caused confusion for folks who want to run it manually, which ends up happening fairly often.